packages/stylelint-config/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "lint": "eslint .",
-    "test": "jest"
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest"
```

Comment:
This was one strange caveat, which I presume is b/c something changed to ESM. So just following the guide here:
https://jestjs.io/docs/ecmascript-modules

Reply:
Yeah the Prettier v3 API is ESM-only now.

Reply:
Also wait are you sure you still need this flag? All the files (including tests) in this subpackage use CJS syntax and the `package.json` still has a `main` field 🤔

Reply:
I can't really explain why, other than the fact that `jest` somehow is aware that the internal `prettier` package is classified as an ESM-only package, but yes. We still do need to run `node` in experimental mode in order for it to support modules :( 

![image](https://github.com/readmeio/standards/assets/751279/d5f8f677-0bbd-488d-af23-abc237e1b2a8)